### ISSUE TITLE
Private Sigstore requires online verification with Rekor

### ISSUE THREAD

**Issue by adityasaky** [gittuf/gittuf#609]
### Add a description

When using a private Sigstore instance bootstrapped using env vars, we don't have a way to identify the timestamp from which point the Rekor instance is trusted. Thus, offline verification of the inclusion promise / proof fails and we fall back to online verification.

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by Horiodino** [comment]
I don't think this issue is necessary now  since there's only offline verification.

**Comment by adityasaky** [comment]
Well, we still need to address this in gittuf :)
